client/containers/search_bar.js

```diff
@@ -24,11 +24,15 @@ class SearchBar extends Component {
 	// initiates the fetchByZip action to perform the search.
 	onFormSubmit(event) {
 		event.preventDefault();
-
 		this.props.fetchByZip( this.state.searchTerm );
+		this.props.showList();
 		this.setState( { searchTerm: '' } );
 	}
 
+	onButtonClick(event) {
+		// this.props.showList();
+	}
+
```

Comment:
added notes - for the eventual geo-location option (not functioning at the moment...)